docs/monitoring/grafanaorganization-monitoring.md

```diff
@@ -0,0 +1,74 @@
+# GrafanaOrganization Monitoring Guide
+
+This guide explains how to monitor GrafanaOrganization custom resources deployed in your Kubernetes cluster using the observability-operator's built-in metrics and monitoring capabilities.
```

Comment:
Who is this doc for ? Maintainers of this project or external users ?

I feel strange that we are talking about general "how to monitor" grafana orgs, while later on we are very specific on the implementations details of how its done here.


Also this doc overlap with the previous one which describe how to **use** the metrics while this doc explains the how to **instrument** the code to produce metrics, concerns should be clearly separated here.